backend/scripts/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Runs blnd-tender tests.
```

Comment:
Pls update, there is no `blnd-tender`.

Reply:
If it was copy-pasted from somewhere, most likely their license requires you to preserve the copyright notice and the license.

Reply:
I don't know the original place of copy-pasting it\
It was in disciplina/agora/blnd and so on (maybe before disciplina there also were some projects)\
So, from where I need to take it?